code/game/objects/items/melee/baton.dm

```diff
@@ -131,6 +131,11 @@
 
 	if(check_parried(target, user))
 		return BATON_ATTACK_DONE
+	if(ishuman(user))
+		var/mob/living/carbon/human/potential_chunky_finger_human = user
+		if(potential_chunky_finger_human.check_chunky_fingers())
+			potential_chunky_finger_human.balloon_alert(potential_chunky_finger_human, "fingers are too big!")
```

Comment:
What happens if this is just `balloon_alert(`? Does it come from the in-hand item? If so that's a shame

Reply:
could target it from teh user instead

Reply:
then this should just be balloon_alert

Reply:
based on my understanding of balloon_alert(), it comes from what it's called on and is shown to the person in arg1